Template10 (Library)/Controls/HamburgerMenu.xaml

```diff
@@ -247,12 +364,10 @@
             </Interactivity:Interaction.Behaviors>
         </ContentControl>
 
-        <SplitView x:Name="ShellSplitView" Grid.Column="0" AutomationProperties.Name="Jerry Nixon"
```

Comment:
Wow. Thanks. That was an oversight when I was doing some accessibility testing. It's also, by the way, the root of a bug on the backlog in the SplitView for accessibility. There's an extra Pane that can't be labeled in the tree. But this was supposed to be removed. Thanks. It was invisible though to the readers.